openshift/performance/patches/002-go-mod.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/go.mod b/go.mod
```

Comment:
Hm I feel like this is probably not super stable (as upstream go.mod changes quite often). Maybe it would be easier to just "add" elasticsearch using a `go get` command in the script?